http/src/main/resources/leo.conf

```diff
@@ -202,6 +202,8 @@ azure {
     instrumentation-enabled = ${?CROMWELL_RUNNER_APP_INSTRUMENTATION_ENABLED}
     enabled = ${?CROMWELL_RUNNER_APP_ENABLED}
     ecm-base-uri = ${?ECM_URL}
+    bard-base-uri = ${?BARD_URL}
```

Comment:
This looks like it is not enabled for Prod or BEEs, just checking that this is the expected behavior coming from leonardo helm values

Reply:
@LizBaldo oh i didnt actually realize that, is it alright to add these variables to prod?